src/index.html

```diff
@@ -11,7 +11,171 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <script
+      type="text/javascript"
+      src="./main.js">
+    </script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="POST"
+    onsubmit="onSubmit()">
```

Comment:
Если у тега больше двух атрибутов - пиши каждый с новой строки
```suggestion
    <form
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="POST"
      onsubmit="onSubmit()"
    >
```